.github/CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ Contributions to the code are even more welcome ;)
 If you'd like to write some code for nf-core/epitopeprediction, the standard workflow is as follows:
 
 1. Check that there isn't already an issue about your idea in the [nf-core/epitopeprediction issues](https://github.com/nf-core/epitopeprediction/issues) to avoid duplicating work
-    * If there isn't one already, please create one so that others know you're working on this
+    * If there isn't one, please create one so that others know you're working on this
```

Comment:
```suggestion
    * If there isn't one already, please create one so that others know you're working on this
```

Reply:
Editing this file is what causes the nf-core lint tests to fail, it should be kept like the template :)